### PR TITLE
Include <cmeow> instead of <meow.h>, Part2

### PR DESCRIPTION
**Pull request by fsb4000**
While reviewing #2851 I found that `tests/std/tests/Dev11_0836436_get_time/test.cpp` includes `#include <assert.h>` and other .h files.
I recall that we already replaced `<meow.h>` to `<cmeow>`. I found the PR: #1405

There are still some places where we include `<meow.h>`:
`tr1 tests`
`benchmarks\inc\xoshiro.hpp` (because `Written in 2018 by David Blackman and Sebastiano Vigna (vigna@acm.org)`)
`__msvc_all_public_headers.hpp`
`<cmeow>`
`tests\std\include\force_include.hpp` (because Part of the LLVM Project)
tests with `// LLVM SOURCES BEGIN` (`tests\std\tests\P0088R3_variant\test.cpp` for example)

### CODE REVIEW

**Review by strega-nil-ms — CHANGES_REQUESTED** [review]
I don't love that the namespaces are changed in this PR, given it's unnecessary AFAIK; additionally, I'm personally at least very much against using `std::` to access C functions and types.

File: stl/src/xxxprec.hpp
Comment (frederick-vs-ja) [reply stl/src/xxxprec.hpp:34]: As a big fan of `std::`-qualifying of C standard library functions, I'd say that this seems consistent with the expected usage of WG21-P2465.
FWIW, when using standard library modules, only `std::printf` will be accessible if only `import std;` is used, while `import std.compat;` should be used in order to use `::printf`.

Comment (fsb4000) [reply stl/src/xxxprec.hpp:34]: Well, I think about it little more: https://gcc.godbolt.org/z/ThWh9jx5T
both `libc++` and `libstdc++` provides both `meow` and `std::meow` when we include `<cmeow>`.
So I think I will remove `std::` and apply the same changing for `test\std\include` and `benchmarks`.

**Comment by fsb4000** [comment]
> clang-format aligning = default/= delete, no manual changes.

oops, I already updated `clang` to 14.
I will merge main and restart pipeline after the toolset update.

**Review by strega-nil-ms — APPROVED** [review]
thanks!

File: tests/std/tests/P1502R1_standard_library_header_units/test.cpp
Comment (StephanTLavavej) [inline tests/std/tests/P1502R1_standard_library_header_units/test.cpp:96]: I'm somewhat concerned about this change, as the Header Units test is special. While the test is continuing to pass, now this is classic-`#include`-ing `<yvals_core.h>` indirectly, after having imported it (indirectly) as a deduplicated header unit. Having the `import`s be the only source of STL machinery is cleaner for the purposes of this test. I would prefer to undo this change and comment the `<meow.h>` lines as being special.

**Review by StephanTLavavej — COMMENTED** [review]
Thanks, this looks good. I've verified that everything that should be changed, is being changed with no missed occurrences, except for the exclusions as noted in the PR description.

As mentioned in a comment, I'm nervous about changing the header units test here - I think we should go back to including the UCRT headers, with added comments.

My last concern is that this is an extremely large PR in terms of the number of files being edited, so we should try to be extra clean about "nice to have" changes that are not strictly necessary for the central intent of the PR. There are a small number of namespace changes, and two of those files were missing changes, which is easily missed in such a large PR.

As much as I love `using namespace std;` in tests (I prefer for all tests in `tests/std` to use using-directives, unless they are one of the unusual ones that is specifically concerned with namespaces like the UDL test), and while we could add additional changes to patch the affected files, I would strongly prefer to remove these changes from the PR so that it's completely focused on the `<cmeow>` transformation - easily verified with `git diff -U0` and `grep` filtering.

I've verified and will push changes. With this, I am confident that the PR shouldn't disrupt tests.

**Review by StephanTLavavej — APPROVED** [review]

**Comment by StephanTLavavej** [comment]
@strega-nil-ms FYI, I pushed changes after you approved.

**Comment by strega-nil-ms** [comment]
@StephanTLavavej's changes LGTM! Thanks.

**Comment by StephanTLavavej** [comment]
I'm mirroring this to the MSVC-internal repo - please notify me if any further changes are pushed.

**Comment by StephanTLavavej** [comment]
I've pushed a merge with `main` to resolve a simple merge conflict with #2125 in `<charconv>` (delete/modify). Accepted delete, as #2125 pre-emptively resolved this conflict in its moved code.

**Review by StephanTLavavej — APPROVED** [review]

**Comment by StephanTLavavej** [comment]
cthanks cfor cthis cconsistency cimprovement! :joy_cat: :tada: :zany_face:

**Pull request merged**
